pontoon/insights/static/js/insights.js

```diff
@@ -28,19 +28,19 @@ var Pontoon = (function (my) {
         }
 
         const colors = [
-          '#fff',
-          '#8074a8',
-          '#c6c1f0',
-          '#c46487',
-          '#ffbed1',
-          '#9c9290',
-          '#c5bfbe',
-          '#9b93c9',
-          '#ddb5d5',
-          '#7c7270',
-          '#f498b6',
-          '#b173a0',
-          '#c799bc',
+          'getComputedStyle(document.documentElement).getPropertyValue("--white-1").trim()',
```

Comment:
We should remove the outer quotes, otherwise http://localhost:8000/insights/ doesn't really use these colors.